vllm/engine/llm_engine.py

```diff
@@ -1436,7 +1436,8 @@ def _process_model_outputs(self,
         # LLMEngine/AsyncLLMEngine directly
         if is_async:
             # Log stats.
-            self.do_log_stats(scheduler_outputs, outputs, finished_before)
+            self.do_log_stats(scheduler_outputs, outputs, finished_before,
+                              skip)
```

Comment:
the format looks a little bit weird. Did you run `bash format.sh`?

Reply:
this is just a cherry-pick of commit [68210201099e6ce1c0a1453633c77fc0185af488](https://github.com/aibrix/vllm/commit/68210201099e6ce1c0a1453633c77fc0185af488) from the main branch, I didn't make any changes in order to make future rebase easier.

Reply:
lgtm, just feel a little bit weird on the format